core/tests/protractor_utils/ContributorDashboardPage.js

```diff
@@ -122,6 +122,9 @@ var ContributorDashboardPage = function() {
 
   this.expectNumberOfOpportunitiesToBe = async function(number) {
     var opportunityCount = (await _getOpportunityElements()).length;
+    await waitFor.visibilityOf(
+      opportunityCount,
+      'Opportunity Count taking too long to appear');
```

Comment:
We don't need this because `opportunityCount` is just a number, not an element. Notice that it's the value of the `.length` attribute